bezier/src/quadratic_bezier.rs

```diff
@@ -246,6 +247,16 @@ impl QuadraticBezierSegment {
 
         return rect(min_x, min_y, max_x - min_x, max_y - min_y);
     }
+
+    /// Returns the smallest rectangle the curve is contained in
+    pub fn minimum_bounding_rect(&self) -> Rect {
+        let min_x = self.from.x.min(self.sample(self.find_x_minimum()).x).min(self.to.x);
```

Comment:
Actually, the find_x_inflection() method returns 

> the x inflection point or None if this curve is x-monotone

So I guess I could write ` let min_x = self.from.x.min(self.sample(self.find_x_minimum()).x) ` instead of also comparing with "self.to.x". What d o you think ? :)